src/test/java/no/unit/nva/model/PublicationTest.java

```diff
@@ -60,14 +60,11 @@ public void updatingDoiStatusSuccessfullyChangesToValidNewDoiStatus()
     }
 
     @Test
-    public void updatingDoiRequestStatusWithInvalidTransitionRequestThenThrowsIllegalArgumentException()
+    public void updatingDoiRequestWithInvalidRequestedStatusChangeThenThrowsIllegalArgumentException()
         throws InvalidIssnException, MalformedContributorException {
         var publication = generatePublicationWithRejectedDoiRequestStatus();
 
-        var actualException = assertThrows(IllegalArgumentException.class,
-            () -> publication.updateDoiRequestStatus(REQUESTED));
-        assertThat(actualException.getMessage(),
-            is(equalTo(String.format(ERROR_MESSAGE_NOT_ALLOWED_TO_CHANGE_STATUS_FROM_S_TO_S, REJECTED, REQUESTED))));
+        assertThrows(IllegalArgumentException.class, () -> publication.updateDoiRequestStatus(REQUESTED));
```

Comment:
Maybe I did not explain it correctly. When we throw an Exception we should also verify that there is some kind of information sent to the caller with the Exception. It has happened in the past that a test was passing because it was expecting a Runtime exception and it passed but the thrown Exception was not thrown where we were expecting to be thrown. That is why we decided to test the expected message as well. 

Another thing is that when we throw an Exception we have some expectations from the message. For example here the initial and the desired status. This gives an insight to the user why their request failed. If i hardcode a message then whenever I correct the message (typo, different expressions, additional information etc.) i will have to correct it multiple tests which are not necessarily insterested in those new additions.